packages/collector/test/uncaught/uncaught_exceptions_test.js

```diff
@@ -142,7 +142,7 @@ mochaSuiteFn('uncaught exceptions', function() {
               expect(serverAcceptedAnotherResponse, "Unexpected response, server shouldn't have accepted another call.")
                 .to.be.false;
               expect(errorFromSecondHttpRequest).to.exist;
-              expect(errorFromSecondHttpRequest.message).to.equal('Error: read ECONNRESET');
+              expect(errorFromSecondHttpRequest.message).to.match(/Error: read ECONNRESET|Error: connect ECONNREFUSED/);
```

Comment:
Gnaah. I'm afraid we need to cover one more error.

https://app.circleci.com/pipelines/github/instana/nodejs-sensor/1327/workflows/d8572e80-f532-4f4b-95d5-d9238d1f225f/jobs/5595

```
expected 'Error: socket hang up' to match /Error: read ECONNRESET|Error: connect ECONNREFUSED/
AssertionError: expected 'Error: socket hang up' to match /Error: read ECONNRESET|Error: connect ECONNREFUSED/
    at agentControls.getEvents.then.events (test/uncaught/uncaught_exceptions_test.js:145:61)
    at tryCatcher (/home/circleci/repo/node_modules/bluebird/js/release/util.js:16:23)
```

Or, alternatively, we remove the expectation for the _content_ of the error message. Now that I think about it, I actually think it is overspecific. I vote to remove line 145 completely.